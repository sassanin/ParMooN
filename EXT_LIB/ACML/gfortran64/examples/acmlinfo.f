*     ACMLINFO Example Program Text
*
*     ACML version 1.0 Copyright AMD,NAG 2003
*
C     .. External Subroutines ..
      EXTERNAL         ACMLINFO
C     .. Executable Statements ..
*
      CALL ACMLINFO()
      STOP
      END
