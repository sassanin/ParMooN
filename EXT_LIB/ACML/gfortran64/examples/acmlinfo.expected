ACML (AMD Core Math Library) version 2.0.1  (Fri Jun 11 09:31:42 BST 2004)
Copyright AMD,NAG 2004
Built using Fortran compiler: GNU Fortran (GCC) 3.3 20030312 (prerelease) (SuSE Linux)
   with flags: -Wall -W -Wno-uninitialized -Wimplicit -fPIC -fno-second-underscore -m64 -DIS_64BIT -O3
 and C compiler: gcc (GCC) 3.3 20030312 (prerelease) (SuSE Linux)
   with flags: -Wall -W -Wno-unused-parameter -Wstrict-prototypes -Wwrite-strings -D_GNU_SOURCE -D_ISOC99_SOURCE -fPIC -m64 -DIS_64BIT -O3
